src/test/core/Config_test.cpp

```diff
@@ -423,8 +429,9 @@ port_wss_admin
             // read from file absolute path
             auto const cwd = current_path ();
             path const dataDirRel ("test_data_dir");
-            path const dataDirAbs (cwd / path ("test_db") / dataDirRel);
-            detail::RippledCfgGuard g (*this, "test_db", dataDirAbs.string (), "");
+            path const dataDirAbs (cwd / path ("test_db00") / dataDirRel);
+            detail::RippledCfgGuard g (*this, "test_db00",
```

Comment:
There's a dependency loop because `dataDirAbs` is a param to create `g`, but I am pushing a nice solution.